common/cluster/metadata.go

```diff
@@ -355,7 +365,7 @@ func (m *metadataImpl) refreshLoop(ctx context.Context) error {
 			for err := m.refreshClusterMetadata(ctx); err != nil; err = m.refreshClusterMetadata(ctx) {
 				m.logger.Error("Error refreshing remote cluster metadata", tag.Error(err))
 				select {
-				case <-time.After(refreshFailureInterval):
+				case <-time.After(m.refreshDuration() / 2):
```

Comment:
why /2?

Reply:
arbitrary number. I feel it just need one refresh internal config. No need to pass in a failure interval. But I am open to this.